tests/integration/test_files.py

```diff
@@ -221,7 +221,9 @@ def test_files_api_upload_download(ucws, random):
             f = io.BytesIO(b"some text data")
             target_file = f'/Volumes/main/{schema}/{volume}/filesit-{random()}.txt'
             w.files.upload(target_file, f)
-
+            # TODO: Enable after generating with the latest spec
```

Comment:
Added to run integration tests with the master sha. I will uncomment it later today during the release.